Frameworks/Foundation/NSOperationQueue.mm

```diff
@@ -330,132 +424,43 @@ - (void)addOperations:(id)operations waitUntilFinished:(BOOL)wait {
 */
 - (void)setMaxConcurrentOperationCount:(NSInteger)count {
     UNIMPLEMENTED();
-    priv->_maxConcurrentOperationCount = count;
+    _priv->SetMaxConcurrentOperationCount(count);
```

Comment:
swell.

---
In reply to: [95453130](https://github.com/Microsoft/WinObjC/pull/1676#discussion_r95453130) [](ancestors = 95453130)